CHANGELOG.md

```diff
@@ -1,16 +1,27 @@
-# Changlog
+# Changelog
+All notable changes to this project will be documented in this file.
 
-## [New Releases here](https://github.com/kickstarter/rack-attack/releases)
+The format is based on [Keep a Changelog](http://keepachangelog.com/)
+and this project adheres to [Semantic Versioning](http://semver.org/).
 
-This file is kept for historical documentation.
 
-## v5.0.0.beta1 4 July 2016
+## v5.1.0 - 10 Mar 2018
+
+  - Fixes edge case bug when using ruby 2.5 and redis #253 (#271)
+  - Throws errors with better semantics when missing or misconfigured store caches to aid in developers debugging their configs (#274)
+  - Removed legacy code that was originally intended for Rails 3 apps (#264)
+
+## v5.0.1 - 11 Aug 2016
+
+  - Fixes arguments passed to deprecated internal methods. (#198)
+
+## v5.0.0 - 4 July 2016
```

Comment:
`5.0.0` was out Aug 9, 2016 actually.

Reply:
You are right. I left the beta release date.

Reply:
Fixed in #3deb14a